zerver/tests/test_custom_profile_data.py

```diff
@@ -434,9 +435,16 @@ def test_update_profile_data_successfully(self) -> None:
 
         iago = self.example_user('iago')
         expected_value = {f['id']: f['value'] for f in data}
+        expected_rendered_value = {}  # type: Dict[Union[int, float, str, None], Union[str, None]]
```

Comment:
Shouldn't it?
```
expected_rendered_value = {}  # type: Dict[int, Union[str, None]]
```
field id will always be an integer value.

Reply:
I also thought the same, however when I tried it out that way, I received a mypy error:
> zerver/tests/test_custom_profile_data.py:447: error: Invalid index type "Union[int, float, str, None]" for "Dict[int, Optional[str]]"; expected type "int"

Reply:
oh, mypy error. Let it be as it is, not big issue.

Reply:
Ok :+1: .